### PR TITLE
Add Procfile for running the app on host

### PR DESCRIPTION
**Pull request by matharden**

### CODE REVIEW

**Comment by r4vi** [comment]
@matharden you might want to copy it over

**Comment by matharden** [comment]
FAIL!

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/matharden/Dropbox/Code/localsh/mtp-cashbook.procfile
```
Comment (SteveMarshall) [inline Procfile]: Do I have to be you to run this?

Comment (matharden) [inline Procfile]: lol - yeah!

**Comment by SteveMarshall** [comment]
I'll look at this with you on Monday, @matharden .

**Comment by SteveMarshall** [comment]
As discussed, we aren't going to use this. Instead, can you open a PR to add `RUN_FLAGS` to `make run`?

**Pull request closed without merging**